connectivity/FEATURE_BLE/source/common/ble_trace_helpers.cpp

```diff
@@ -522,7 +522,7 @@ const char *to_string(const UUID& uuid)
     static constexpr const char *HEX = "0123456789ABCDEF";
     static constexpr const size_t HYPHENS_DELIMITER_COUNT = 4;
 
-    static char string_buffer[ROW_COUNT][UUID::LENGTH_OF_LONG_UUID + HYPHENS_DELIMITER_COUNT];
+    static char string_buffer[ROW_COUNT][2*UUID::LENGTH_OF_LONG_UUID + HYPHENS_DELIMITER_COUNT + 1];
```

Comment:
It's `2*` because each byte needs up to two hex digits. It's `+ 1` for the terminating `NUL`.

If this is correct, could you please add a comment in the code to explain?

Style: We might also need to do `2 * UUID::LENGTH_OF_LONG_UUID` instead of `2*UUID::LENGTH_OF_LONG_UUID` to pass the style checker.

Reply:
@Patater That is indeed correct. I added a comment explaining it in the code and updated the style: 7ac8ab92ea56c5012b145c16acfabeb52f3af722

I will look into adding a regression test for this tomorrow.